model_public_session/controllers/main.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import json
+from openerp import http
+from openerp.http import request
+from openerp.addons.website.controllers.main import Website
+
+
+class Website(Website):
+
+    @http.route(
+        '/website/field_autocomplete/<string:model>',
+        type='http',
+        auth='public',
+        methods=['GET'],
+        website=True,
+    )
+    def _get_field_autocomplete(self, model, **kwargs):
```

Comment:
When is this controller method used?
I checked the js code for the autocomplete, I see it use `search_read` and can't find any reference to an `/website/field_autocomplete/` URL.